loan-pattern/src/main/java/com/iluwatar/loan/pattern/App.java

```diff
@@ -0,0 +1,41 @@
+package com.iluwatar.loan.pattern;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class App {
```

Comment:
Here, above the app, please add an explanation of the pattern and how this example implements it